OpenRA.Mods.Common/Traits/Buildings/Refinery.cs

```diff
@@ -29,6 +29,9 @@ public class RefineryInfo : TraitInfo, Requires<WithSpriteBodyInfo>, IAcceptReso
 		[Desc("Docking cell relative to top-left cell.")]
 		public readonly CVec DockOffset = CVec.Zero;
 
+		[Desc("If harvester in an aircraft does it need to land to unload resources.")]
+		public readonly bool RequireAircraftToLand = true;
```

Comment:
This is not even used here and odd for the default maps. Maybe use an existing trait from `Aircraft` or put this onto an `AircraftHarvester`.

Reply:
We want this to be a choice on a per dock basis, so it needs to stay on `Refinery` until `Dock` class is added